toolchains/workspace-pnpm/macros.bzl

```diff
@@ -1165,9 +1165,12 @@ _jest_test = rule(
             default = False,
             doc = "Run all tests serially in the current process"
         ),
-        "env_file": attrs.option(
-            attrs.string(),
-            doc = """File name and relative path for env variables required.""",
+        "app_env": attrs.dict(
```

Comment:
We have `build_env` (for things like next build) and `env` (for test-runner like jest) attributes in other places. I chose to go with `app_env` (for application code) here but open to alternatives